src/plot.jl

```diff
@@ -0,0 +1,448 @@
+@info "Loading DataFlowTasks plot utilities"
+
+using .GLMakie
```

Comment:
Because we had to call "using GraphViz" before compiling this code, using GraphViz.funcname was sufficient, so it didn't complained.
I added the using .GraphViz for consistency.